docs/connectors/mysql.asciidoc

```diff
@@ -895,7 +895,52 @@ Other data type mappings are described in the following sections.
 [[temporal-values]]
 ==== Temporal values
 
-Other than MySQL's `TIMESTAMP` data type, the other MySQL temporal types depend on the value of the `time.precision.mode` configuration property. When `time.precision.mode` configuration property is set to `adaptive` (the default), then the connector will determine the literal type and semantic type for the temporal types based on the column's data type definition so that events _exactly_ represents the values in the database:
+Other than MySQL's `TIMESTAMP` data type, the other MySQL temporal types depend on the value of the `time.precision.mode` configuration property.
+
+[NOTE]
+====
+As of Debezium 0.7 `adaptive_time_microseconds` mode was introduced and is the default `time.precision.mode` for the MySQL connector. Mode `adaptive` was marked as deprecated.
+====
+
+[WARNING]
+====
+Debezium can only capture positive TIME field values in the range of `00:00:00.000000` to `23:59:59.999999` when `time.precision.mode` is set to `adaptive`, and `00:00:00.000` to `23:59:59.999` when `time.precision.mode` is set to `connect`.
```

Comment:
I'd rever these sentences to make clear what's cause and what's consequence:

> When `time.precision.mode` is set to `adaptive`, only positive TIME field values in the range of `00:00:00.000000` to `23:59:59.999999` can be captured correctly. When `time.precision.mode` is set to `connect`, only values in the range of `00:00:00.000` to `23:59:59.999` can be handled.

Reply:
s/todo/done/